### DISCUSSION TITLE
Which framework to use graphql-java or graphql-java-kickstart

### DISCUSSION THREAD

**Discussion by VishnuGupta16** [graphql-java/graphql-java#2259] [Q&A] [answered]
Hi All,

I am currently exploring graphql implementation for my project, I am using Spring Boot 2.1.8-RELEASE.

I am confused as to which one to use between graphql-java or graphql-java-kickstart. 

I am looking for regular updates for new features and bug fixes.

**Comment by ooga** [comment, marked as answer]
Hi,
it's not really comparable because they have different scope.

graphql-java is a core library to work with graphql
graphql-java-kickstart is a library which use graphql-java

If you want to use a library compatible with spring boot you may have a look at
| Maintainer | Project | Spring boot |
| - | - | - |
| Community | [graphql-java-kickstart](https://www.graphql-java-kickstart.com/spring-boot/) | mvc and webflux |
| Netflix | [dgs](https://netflix.github.io/dgs/) | mvc only, webflux is planned in a near future |
| Expedia | [graphql-kotlin](https://expediagroup.github.io/graphql-kotlin/) | webflux only |
